src/Sylius/Behat/Resources/config/services/contexts/cli.xml

```diff
@@ -17,11 +17,14 @@
 
         <service id="Sylius\Behat\Context\Cli\ShowingAvailablePluginsContext">
             <argument type="service" id="__symfony__.kernel" />
+            <argument type="service" id="__symfony__.Sylius\Bundle\CoreBundle\Command\SetupCommand" />
```

Comment:
We don't need `__symfony__` prefixes since `1.4` (to which branch this PR should be rebased)